### ISSUE TITLE
[idea] Is there any way we can build this project without using snap?

### ISSUE THREAD

**Issue by mohamadali-halwani** [ropoko/cli-dictionary#11]
**Greetings Dear ropoko,**
**I hope this post finds you well.**

**I am trying to run this project using nothing else but python3 and anki. The main issue for me is the circular import statements in main.py**
**I would like to make your dictionary native to python3, without having to use something like snapcraft or flatpak. Some maintainers frown upon using such tools for their own good reasons. For me, it's just that I'm new to this field.**
The issues I am facing with your respected code, are the following errors:
### 1-Although I double checked the file naming and nothing was out of place, I had this error:
```
Traceback (most recent call last):
  File "/home/ignis/projects/1-Work/dictionary-cli/./bin/dictionary-cli.py", line 16, in <module>
    from languages import language
ModuleNotFoundError: No module named 'languages'

```
### 2-Even when I comment-out the import statements like this:
![import statements commented out](https://user-images.githubusercontent.com/68155980/198585999-6fc6a044-f592-4a51-9d14-859ca4708348.png)
### I face the following error:
```
Traceback (most recent call last):
  File "/home/ignis/projects/1-Work/dictionary-cli/./bin/dictionary-cli.py", line 260, in <module>
    main(sys.argv[1], [args])
IndexError: list index out of range
```

To sum up my post:
1. Is there any python3 or python2 libraries I should be focusing on to make this work without snap or flatpak? 
2. Why did you use snapcraft instead of making it a native `pip` package?
3. Do you think it is hard to run this program natively? If so, why?
4. If your time allows, do you think we can cooperate to make this work? Maybe by assigning a new mutual project.
5. If you do not have the time, can you at least help by hinting on where to go from here?

I appreciate your efforts making this program.
Kindly waiting,
MohamadAli Halwani

**Comment by ropoko** [comment]
hey thanks for the interest!

to the first thing you pointed out, you can try using the branch `feature/refactoring`.

for the other questions: 

1 - this project basically only use requests (to make http requests)

2 - I developed for snap because I wanted to try and learn about it as I saw snap as a good thing and I can't imagine people installing an app from **pip**, because it is a package manager, but I will reconsider this.

3 - the program can be easily run natively, the explanation is on topic 2

4 - absolutely, start creating a pull request in what you think could be better
